src/ocamlorg_frontend/components/learn_components.eml

```diff
@@ -154,31 +154,46 @@ let right_books_content_block books ~title ~(link : link) =
   </div>
 
 let contribute_footer ~href =
-  <div class="pt-10 mt-16 border-t border-slate-200 lg:grid lg:grid-cols-3 gap-4 text-slate-500">
-    <div class="lg:col-span-1">
-      <h2 class="mt-0 mb-4 text-base font-semibold leading-6 lg:text-base">Help Improve Our Documentation</h2>
-      <p class="text-sm mt-0 mb-4 max-w-xs">All OCaml docs are open source. See something that's wrong or unclear? Submit a pull request.</p>
-      <a
-        class="inline-block relative py-1 px-4 font-medium no-underline align-middle bg-transparent rounded-md border border-solid appearance-none cursor-pointer select-none whitespace-no-wrap hover:no-underline"
-        href="<%s href %>"
-      >
-        <%s! Icons.github_pull_request "inline-block overflow-visible mr-0 align-text-bottom select-none" %>
-        Contribute</a
-      >
+  <div class="my-5 border-y border-gray-200 pt-5">
+    <div class="mb-10 mt-4">
+      <h2 class="mb-4 mt-0 text-lg font-bold leading-6">Also Read:</h2>
+      <ul class="list-disc marker:text-orange-500">
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+      </ul>
     </div>
-    <div class="mt-4 lg:mt-0 lg:col-span-1 lg:col-start-3">
-      <h2 class="mt-0 mb-4 text-base font-semibold leading-6 lg:text-base">Still need help?</h2>
-      <div class="text-sm mb-2">
-        <a href="https://discuss.ocaml.org/" class="font-semibold no-underline bg-transparent cursor-pointer hover:underline">
-          <%s! Icons.community "inline h-4 w-4" %>
-          Ask the OCaml Community</a
-        >
+    <div class="grid-cols-2 gap-4 border-t border-gray-200 sm:grid">
+      <div class="border-b border-gray-200 px-6 pb-3 sm:border-b-0 sm:border-r">
```

Comment:
```suggestion
    <div class="grid-cols-2 gap-4 border-t border-gray-200 md:grid">
      <div class="border-b border-gray-200 px-6 pb-3 md:border-b-0 md:border-r">
```

Let's go with the `md` breakpoint, because the line breaks on the smallest `sm` screen are breaking the alignment a bit.

![Screenshot 2023-10-13 at 13-20-32 Get Up and Running With OCaml · OCaml Tutorials](https://github.com/ocaml/ocaml.org/assets/6594573/02438f90-3095-4fe4-85ad-e9e2336fa4b0)
vs
![Screenshot 2023-10-13 at 13-21-04 Get Up and Running With OCaml · OCaml Tutorials](https://github.com/ocaml/ocaml.org/assets/6594573/e35922bb-6c95-4cd0-b6e4-cae8a868f48a)